Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/inbound/deferred/response/TestInboundPatientDiscoveryDeferredResponse.java

```diff
@@ -37,8 +37,16 @@
  */
 public class TestInboundPatientDiscoveryDeferredResponse implements InboundPatientDiscoveryDeferredResponse{
```

Comment:
This is used by PatientDiscovery_10 Gateway module to test the standard and Passthrough classes are instantiated to test applicationContext.xml . If I remove this class from here that's going to fail.